paddle/fluid/memory/allocation/allocator_facade.cc

```diff
@@ -37,7 +37,7 @@
 #endif
 
 DEFINE_int64(
-    gpu_allocator_retry_time, 0,
+    gpu_allocator_retry_time, 100,
```

Comment:
Why do you set 100 here?

Reply:
I am not sure what this value should be. TensorFlow sets it to be 10 seconds, but I think it is too large...